tools/deployJournals.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+const path_     = require('path');
+const fs_       = require('fs');
+const path = path_.posix;
+
+require('../src/foam_node.js');
+
+var [argv, X, flags] = require('./processArgs.js')(
+  '',
+  { version: '', license: '', pom: 'pom' },
+  { debug: true, java: false, web: true }
+);
+
+foam.require(X.pom, false, true);
+
+const outPath = 'runtime/journals';
+
+if ( ! fs_.existsSync(outPath) ) {
+    fs_.mkdirSync(outPath, { recursive: true });
+}
+
+async function findJournals({ jrls, srcPath, jrlName }) {
+    const walker = foam.util.filesystem.FileWalker.create();
+    walker.files.sub((_1, _2, info) => {
+        for ( const fileInfo of info.files ) {
+            if ( foam.poms[fileInfo.path] != undefined ) continue;
```

Comment:
this will ignore all paths immediately inside `srcPath`, but not directories within those directories (which is why it still gets some journal entries).

When #1680 is merged, this should be replaced with:
```javascript
if ( fileInfo.path !== srcPath && foam.poms[fileInfo.path] != undefined ) walker.skip.pub();
```